File: BarberBooking.API/Filters/PageParams.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using BarberBooking.API.Models;

namespace BarberBooking.API.Filters
{
    public class PageParams
    {
        public int? Page { get; set; }
        public int? PageSize { get; set; }
    }
}

File: barber_booking_app/lib/models/params/page_params.dart
import 'package:barber_booking_app/models/base/json_serializable.dart';

class PageParams extends JsonSerializable {
  int? Page;
  int? PageSize;
  PageParams({this.Page, this.PageSize});
  
  @override
  Map<String, dynamic> toJson() {
      return{
        'Page':Page,
        'PageSize':PageSize
      };
  }
}